jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```
@@ -0,0 +1,497 @@
+    "import json\n",
+    "import math"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auth = HTTPBasicAuth(PL_API_KEY, '')\n",
+    "session = requests.Session()\n",
+    "\n",
+    "# Authenticate\n",
+    "session.auth = (PL_API_KEY, '')"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Search for the mosaic ID to get its quad IDs\n",
+    "\n",
+    "Here, we are using the full name of the mosaic, but with the `name__contains` field, you can use a partial name based on the names of the mosaics available through your account.\n",
+    "\n",
+    "Declare the search string for `requests.get()`. You will use the dictionary you create with the mosaic name to pass to the requests.get() function.\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This code sets the URL and headers for an HTTP request to Planet's Basemap API and sends a GET request with the specified search parameters. If the response returns the correct status, the JSON response is loaded into a Python dictionary with mosaics information from which we'll extract quad ID information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BASEMAP_API_URL = 'https://api.planet.com/basemaps/v1/mosaics'\n",
+    "headers = {'content-type': 'application/json'}\n",
+    "\n",
+    "basemap_params = {\n",
+    "    'name__contains': 'point_reyes_rolling_normalized_2020-04-30_mosaic'\n",
+    "}\n",
+    "\n",
+    "basemap_service_response = requests.get(\n",
+    "    url=BASEMAP_API_URL, params=basemap_params, auth=auth, headers=headers)\n",
+    "\n",
+    "\n",
+    "if basemap_service_response.status_code == 204:\n",
+    "    basemaps = basemap_service_response.raise_for_status()\n",
```

This error handling should be different. Status codes like 401 (Forbidden) or 404 (not found) should also raise an error. I'd suggest:

```
if basemap_service_response.status_code < 400:
    basemaps = basemap_service_response.json()
else:
   basemap_service_response.raise_for_status()
```